lib/developer_portal/lib/liquid/drops/message.rb

```diff
@@ -5,7 +5,7 @@ class Message < Drops::Model
 
       allowed_name :message, :messages, :reply
 
-      desc "Returns the ID of the model (depending on controller that is message or message recipient)."
+      desc "Returns the ID of the received or sent message."
```

Comment:
Can we then add a normal code comment here in this case, to make the above known?

Reply:
There is already a comment here: https://github.com/3scale/porta/blob/master/lib/developer_portal/lib/liquid/drops/message.rb#L3

It is really close to that `#id` method :)